src/components/Proposals/FeedbackModule.js

```diff
@@ -101,18 +98,18 @@ function FeedbackModule({ vote, connectedAccount }) {
   )
 }
 
-function Bold({ children }) {
+function Strong({ children }) {
```

Comment:
I'm not sure we need this component, I think the globals are already providing these styles for paragraph and strong tags.

Reply:
We need it because we also need to change font color

Reply:
ah, I thought paragraphs were already defaulting to `surfaceContent`